### ISSUE TITLE
Doesn't seem work as this

### ISSUE THREAD

**Issue by TimLuminesence** [aws-samples/amazon-ivs-auto-captions-web-demo#2]
Hi, 

Tried the code. Doesn't seem work as this. 
We have some questions.
1. Looks like the download address for pcre-8.44.tar.gz is not available, need the following changes.
In ./serverless/stream-server/Dockerfile, need to change. 
wget https://ftp.pcre.org/pub/pcre/pcre-8.44.tar.gz && \
to
wget https://sourceforge.net/projects/pcre/files/pcre/8.44/pcre-8.44.tar.gz && \
    
2. In aws configure, how should we set "Default output format"?

3. With the change 1 which I listed above. When run "bash deploy.sh", do you have any idea what this error means and how can I fix it?
Waiter StackCreateComplete failed: Waiter encountered a terminal failure state: For expression "Stacks[].StackStatus" we matched expected path: "ROLLBACK_COMPLETE" at least once

Generating environment variables file for Player App...node:internal/fs/utils:344
    throw err;
    ^

Error: ENOENT: no such file or directory, open 'stack.json'
    at Object.openSync (node:fs:585:3)
    at Object.readFileSync (node:fs:453:35)
    at Object.<anonymous> (/Users/tianyu/Downloads/amazon-ivs-auto-captions-web-demo-2.0.2/deployment/generate-player-app-env-vars.js:13:33)
    at Module._compile (node:internal/modules/cjs/loader:1101:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: 'stack.json'
}

Thanks a lot!

**Comment by mboulin** [comment]
@TimLuminesence Thanks for creating this issue. We are investigating. I will comment again once a fix has been merged.

**Comment by mboulin** [comment]
@TimLuminesence an updated version of the solution is now available on "main", with that unused dependency removed. Please re-clone the repository and try again.

As for 2/, simply use the default (json).

Please follow the deployment steps to deploy the solution [https://github.com/aws-samples/amazon-ivs-auto-captions-web-demo/blob/main/deployment/README.md](https://github.com/aws-samples/amazon-ivs-auto-captions-web-demo/blob/main/deployment/README.md) 
Please make sure Docker (and its daemon) are running on your machine before starting the deploy process, as the container images will be built locally before being uploaded.

**Comment by TimLuminesence** [comment]
Great! Thank you very much! Let me try it.
Actually, in old version, after I do 1 and set json to 2. It works. Thanks!

**Comment by TimLuminesence** [comment]
@mboulin The new code works!
Thanks you very much!
I notice the delay of captions is a little bit bigger than the old version when I modified it to make it work. Does that expected? Thanks!

**Comment by TimLuminesence** [comment]
@mboulin This demo supports audio is English to captions. If I want to add audio is an other language, like Chinese, to caption. How can I do that? Thanks!

**Comment by mboulin** [comment]
Glad the demo is now working for you!
This demo was built with English (en-US) as the language used for auto-captions.
You can change the default language/locale by editing `LANGUAGE_CODE`, `DEFAULT_LANGUAGE_CODE` and
`MEDIA_SAMPLE_RATE_HERTZ` in the `constant.js` file.

You can see a list of supported languages by Amazon Transcribe here: https://docs.aws.amazon.com/transcribe/latest/dg/supported-languages.html

**Comment by mboulin** [comment]
Since the original issue has been addressed, I am closing this issue. Feel free to open a new issue if needed.

**Comment by TimLuminesence** [comment]
Thank you very much!
